Add-on Documentation/Deployment/Phrase.md

```diff
@@ -18,7 +18,7 @@ Once the add-on has been provisioned the `PHRASE_AUTH_TOKEN` will be available w
 
 ## Installation and Usage
 
-phrase should be integrated in an own staging environment. If you haven't done so already, we recommend you set up a staging environment as explained in the [CloudControl Documentation](https://www.cloudcontrol.com/dev-center/Platform%20Documentation#development-staging-and-production-environments). The staging environment will be the place for your translators to edit your website content.
+phrase should be integrated in an own staging environment. If you haven't done so already, we recommend you set up a staging environment as explained in the [CloudControl Documentation](https://www.cloudcontrol.com/dev-center/platform-documentation#development-staging-and-production-environments). The staging environment will be the place for your translators to edit your website content.

```

Comment:
This links needs to be `/dev-center/platform-documentation#development,-staging-and-production-environments`